tests/llmobs/conftest.py

```diff
@@ -212,3 +215,53 @@ def mock_ragas_evaluator(mock_llmobs_eval_metric_writer, ragas):
     LLMObsMockRagas.return_value = 1.0
     yield RagasFaithfulnessEvaluator
     patcher.stop()
+
+
+@pytest.fixture
+def tracer():
+    return DummyTracer()
+
+
+@pytest.fixture
+def llmobs_env():
+    return {
+        "DD_API_KEY": "<default-not-a-real-key>",
+        "DD_LLMOBS_ML_APP": "unnamed-ml-app",
+    }
+
+
+class TestLLMObsSpanWriter(LLMObsSpanWriter):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.events = []
+
+    def enqueue(self, event):
+        self.events.append(event)
```

Comment:
Nice, overwriting the `enqueue()` method to avoid actually sending the event. You love to see it